### ISSUE TITLE
[Arcs] Spirit of Freedom when there is also a tie with a Member

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#78]
[Spirit of Freedom](https://cards.ledergames.com/card/ARCS-F224):
> You may secure cards from the Court where you are tied for most agents with a Rival who is not a Commonwealth Member.

A question asked https://discord.com/channels/476234833572397056/963112189802201108/1307071662314881055 what happens when there is a three-way tie among the Founder, a member, and a non-member, would the Founder be able to secure?

**Comment by jdyearsley** [comment]
Yes—it fulfills the text as written. It's slightly goofy, but I would thematically justify it as the Founder pressuring the Commonwealth Member to support their claim to the card. I'll add an FAQ entry for this.
